PROJECT

```diff
@@ -0,0 +1,19 @@
+domain: sentinel.io
+repo: sentinel-go-k8s-crd-datasource
+resources:
+- group: datasource
+  kind: CircuitBreakerRules
+  version: v1
+- group: datasource
+  kind: FlowRules
+  version: v1
+- group: datasource
+  kind: HotspotRules
```

Comment:
Maybe HotspotParamFlowRules can be more concrete?

Reply:
Maybe we should keep naming consistent with hotspot module?